source/sass/views/youtube-regrets-2022.scss

```
@@ -0,0 +1,57 @@
+.youtube-regrets-2022 {
+  $font-zilla: "Zilla Slab", sans-serif;
+
+  .recommendations {
+    background-color: $black;
+    padding-top: 85px;
+
+    &__title {
+      font-size: 30px;
+      line-height: calc(34 / 30);
+
+      @media screen and (min-width: $bp-md) {
+        font-size: 52px;
+        line-height: calc(60 / 52);
+      }
+    }
+
+    &__recommendation {
+      font-size: 24px;
+      line-height: calc(30 / 24);
+
+      @media screen and (min-width: $bp-md) {
+        font-size: 28px;
+        line-height: calc(30 / 28);
```

Even though we are reusing a bunch it would be nice to remove this needless math haha

```suggestion
        line-height: 1;
```